android/versioned-abis/expoview-abi40_0_0/build.gradle

```diff
@@ -168,6 +168,5 @@ dependencies {
   // @react-native-community/viewpager
   api 'com.github.troZee:ViewPager2:v1.0.6'
 
-  // expo-payments-stripe
   implementation 'com.stripe:stripe-android:16.10.0'
```

Comment:
Is this still needed in expoview? Isn't it installed automatically by stripe-react-native?

Reply:
😮  I think that's right! good catch

Reply:
Actually no, since we're vendoring the code we need this